.github/workflows/llext.yml

```diff
@@ -34,8 +34,13 @@ jobs:
       - name: llext build
         run: |
           cd workspace && ./sof/zephyr/docker-run.sh /bin/sh -c \
-             'ln -s  /opt/toolchains/zephyr-sdk-*  ~/;
-              west build --board intel_adsp_ace15_mtpm sof/app              \
-              -- -DEXTRA_CFLAGS=-Werror -DEXTRA_CXXFLAGS=-Werror     \
-              -DEXTRA_AFLAGS=-Werror \
-              -DOVERLAY_CONFIG=overlays/mtl/module_overlay.conf'
+             "ln -s  /opt/toolchains/zephyr-sdk-*  ~/;
+              python sof/scripts/xtensa-build-zephyr.py \
+              --no-interactive \
+              --cmake-args=-DEXTRA_CFLAGS=-Werror \
+              --cmake-args=-DEXTRA_CXXFLAGS=-Werror \
+              --cmake-args=-DEXTRA_AFLAGS='-Werror -Wa,--fatal-warnings' \
+              --cmake-args=--warn-uninitialized \
+              --cmake-args=-DCONFIG_OUTPUT_DISASSEMBLY=y \
+              --cmake-args=-DCONFIG_OUTPUT_DISASSEMBLY_WITH_SOURCE=n \
```

Comment:
You don't need `--no-interactive` (no one does TBH)

Since this workflow does not save artefacts, DCONFIG_OUTPUT_DISASSEMBLY is only slowing things down (almost twice slower build time total on some systems)